### DISCUSSION TITLE
How can i have any type off mobile with android

### DISCUSSION THREAD

**Discussion by hassanjalil88** [mishakorzik/AllHackingTools#44] [General]
Please tell me

**Comment by on3do** [comment]
BUG VISUAL

The banner and the action selection interface are disorganized on my device (xiaomi 8 lite)
![Screenshot_2022-11-30-17-19-59-563_com termux 1](https://user-images.githubusercontent.com/119537177/204900209-a7b22f2d-8d49-4551-a93c-999084b6f638.jpg)

**Reply by mishakorzik** [reply]
Its not problem AllHackingTools. Something is wrong with your terminal

**Comment by luna44404** [comment]
Get in touch with a professional hacker ( tech ) to help hack your cell phone. Trusted and verified with quick responds and legit services. They offer services like (Cell phone hack , GPS tracker, Delete criminal records, Retrieve wallet, Retrieve Gm ail, face book, whatsApp, photos and many more...), All these services are done remotely, distance is not a barrier. You can reach out with them on ( SPYRECOVERY36 @ gm ail c om ).

**Reply by Parallax2015** [reply]
I need a hacker who has his own botnet for hire to use on YouTube for video views, likes, comments and subscribers.

**Comment by HORACE123X** [comment]
Few days ago I came online in search of a hacker that I could hire to do a spy job for me, it was very important so I needed a strong hacker for my job. I came across several reviews but I figured peallyss @ gmail com was well praised for doing similar jobs that I needed. I decided to give this hacker on google mail the job, he gave me 4 hours to wait for results and then he came in. I was more surprised when I was informed I could see everything like a website. I'm Grateful , everyone please reach out to him on peallyss @gmail.com

**Reply by Parallax2015** [reply]
I'm looking for a hacker to create a botnet for me. How much did he charge you for his services? And do you pay with cryptocurrency or PayPal ?

**Comment by HORACE123X** [comment]
Hello
  
Thanks for contacting me   
  
I can help you create a botnet and give you full access to it.
  
Kindly give me your details so I can run a system scan.
  
Thanks
  

  

  

  
  
  
  
  
>

**Reply by Parallax2015** [reply]
Thank you for replying, hopefully you can help me and I will pay you if you give me good results.
You ask for my details, but why do you need my details ? You don't need to scan my PC.

**Reply by Parallax2015** [reply]
But first I need to know what you can do, I need high video views between 500k to 600k for each video, many likes over 1k, 100s of comments and regular subscribers. Can you do this ?  My YouTube channel is called Malibu CA TV series.

**Comment by HORACE123X** [comment]
Alright   
  
I would need to link the account to your computer's ip address
  
Creating the account would take 30mins to 1hour   
  
And the charge for this job is $500
  
Let me know if you would wish to proceed   
  
Thanks   
  
You can also reach out to me via ***@***.*** or whatsapp for more effective communication   
  

  

  

  

  
  
  
  
  
>
